js/app/src/api_docs/ApiDocs.svelte

```diff
@@ -144,13 +148,14 @@
 </script>
 
 {#if info}
-	{#if Object.keys(info.named_endpoints).length + Object.keys(info.unnamed_endpoints).length}
+	{#if dependencies.filter((dependency) => dependency.api_name && dependency.show_api).length}
```

Comment:
I think we're writing `dependency.api_name && dependency.show_api` enough times in this file that we can write a helper function in the script tag to improve readability

Reply:
Let me refactor this